.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - grammar.js
+      - src/**
+      - test/**
+      - bindings/**
+      - binding.gyp
+  pull_request:
+    paths:
+      - grammar.js
+      - src/**
+      - test/**
+      - bindings/**
+      - binding.gyp
+
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Test parser
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-14]
```

Comment:
Is there a particular reason this isn't `macos-latest`? I assume the other parsers you looked at avoid it too, did they give a reason?

Reply:
None that I could find. Based on the timestamps, they updated them to `macos-14` [in April](https://github.com/tree-sitter/tree-sitter-cpp/commit/075886a9fb1f56afa1125dd4f0a51038d112bc2c), and GitHub updated `macos-latest` to `macos-14` [in May](https://github.blog/changelog/2024-05-20-actions-upcoming-changes-to-github-hosted-macos-runners/). So potentially a timing thing.

I think we could probably switch it to `macos-latest` to avoid having to tick it up again in the future.